### PR TITLE
As an HP Owner I can browse available hApps, and host and unhost them

### PR DESCRIPTION
**Pull request by robbiecarlton**

### CODE REVIEW

**Review by lorenjohnson — COMMENTED** [review]
@robbiecarlton I've reviewed, and this looks just fine for now. Is there anything besides PR review and UI work holding this back from being merged.

File: src/components/RegisterUser/RegisterUser.connector.test.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    },
+    result: {
+      data: {
+        hostingUser: mockHostingUser
+      }
+    }
+  }
+]
+
+describe('connector', () => {
```
Comment (robbiecarlton) [inline src/components/RegisterUser/RegisterUser.connector.test.js:26]: see comments in `RegisterUser.test.js` below

File: src/components/RegisterUser/RegisterUser.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import React, { useEffect } from 'react'
+import { get } from 'lodash/fp'
+
+export function RegisterUser (props) {
```
Comment (robbiecarlton) [inline src/components/RegisterUser/RegisterUser.js:4]: This component is mostly a placeholder. I expect our auth/login requirements to be more complicated than this. But for now, we have to make sure that the hostingUser is registered as a host or they can't see the available happs.

File: src/components/RegisterUser/RegisterUser.test.js [hunk 3]
```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import RegisterUser from './RegisterUser'
+import { render } from '@testing-library/react'
+
+const childrenText = 'The Children'
+const children = <div>{childrenText}</div>
+
+describe('RegisterUser', () => {
```
Comment (robbiecarlton) [inline src/components/RegisterUser/RegisterUser.test.js:8]: For this component it felt much better to test the component without the connector. This is an unusually complex component (three possible branches!), and I still currently think the integration style is better for simpler components. But this has reduced my confidence in that a little.

For now I think we just keep doing what seems right in a given case, and check back in regularly to see if there's a conclusion to come to.

**Review by lorenjohnson — COMMENTED** [review]
LGTM 👍

**Pull request merged**